### DISCUSSION TITLE
Incorporating armor weight into rolls

### DISCUSSION THREAD

**Discussion by rozzingit** [Arx-Game/arxcode#445] [Ideas]
So there's a variety of armor types folks can use, and the weight of it is taken into consideration when using the automated combat system. However -- no one uses the automated combat system anymore! Except for sparring. In terms of GMed scenes, I don't think it's been used in a long time. Which is kind of sad in terms of having opportunity to reflect the strengths and weaknesses of the different types of armor.

I know that we have a 'Dodge Penalty' being calculated for PCs that shows up in the `combatstats` command, and I wonder if this can be used somehow. My first thought was just having something automatic applied to Dexterity+Dodge checks, but that would be pretty narrow and probably a bit inflexible. So now I'm wondering about something like a `check/armor` switch. It might not only be dodge checks that people want, after all; you could also use it for something like stealth or athletics, and making it a command switch would allow GMs the flexibility to request it or not as the situation warrants. As for the exact calculation to follow...that would be a matter for the experts in terms of the dice system balance and the like, but the idea would be that the larger your Dodge Penalty is, the more your roll would be penalized. Not that it has to be a huge penalty, but I do think it'd be cool to let there be reason for different types of armor and the like. Right now, there's no real code reason to go for anything but top tier metal, because nothing's checking for fatigue, weight, etc.

My only other thought is that there _is_ a potential abuse case I can see wherein someone could quietly remove pieces of armor, roll, and then put them back on. Not sure what the best way to avoid that is. A check that puts a dodge penalty value into plain English? So maybe something like `Aleksei checks dexterity and dodge at normal. Aleksei is successful.` would become `Aleksei checks dexterity and dodge at normal with no encumbrance. Aleksei is successful.` I don't know the right words to use, but it'd at least make it look pretty obvious if a character is geared up in platemail and their check tries to say they've got no weight to their armor.

THAT IS MY THOUGHT.

**Comment by m2abrams15** [comment]
So this is interesting as the combat code already has allowances for reduction to dodge based on Armor type, but as you say, that code is never/rarely used in prps or plot scenes.  there are a lot of intricacies in that code that are not being used in the current system of checks used in prps.

im all for anything that adds depth to the system, and maybe for combat checks the Oldcheck system is better to be used as it allows for more fidelity in the numbers.  Could add a function for a gm to tag a penalty to each person for physical checks  whihc would just be subtracted after the knack addition perhaps?

**Comment by rozzingit** [comment]
I feel like going back to oldcheck for anything runs pretty directly contrary to how the game designers have shifted the game. It's an entirely different dice system with a different bell curve, different probabilities, etc. I don't really think that's a good solution, especially for something as significant as combat.

**Comment by m2abrams15** [comment]
Excellent point! i think the existing armor penalty system is based on the old check system as well. so would need to figure out new numbers.

The other thing to think on, is the highger risk events, armor matters less and less as it efects dmg less and less.  So might this lead to a situation where its actually better to wear LESS armor in higher risk situations (which obviously makes no sense :P )  so i wonder if the value armor can have to incoming harm would need to be adjusted as well?

**Comment by rozzingit** [comment]
Yeah, I purposefully didn't bring up _how_ the numbers should be balanced, because I think only the coders who are in the very thick of it and designing that balance would be able to make that particular call. I just know that there's _some_ sort of numerical value being calculated in the background regarding armor weights and whatnot. I'm sure it can't just be used as-is, but I imagine it could be a basis/starting point for something.
